lib/outdated.js

```diff
@@ -48,6 +48,12 @@ function outdated (args, silent, cb) {
   outdated_(args, dir, {}, 0, function (er, list) {
     if (!list) list = []
     if (er || silent || list.length === 0) return cb(er, list)
+    list.sort(function(a, b) {
+      var aa = a[1].toLowerCase()
+        , bb = b[1].toLowerCase()
+      return aa === bb ? 0
```

Comment:
Good catch – if you wanted to patch this to be

```javascript
var aa = a.toLowerCase()
var bb = b.toLowerCase()
return aa == bb ? 0 : aa < bb ? -1 : 1
```
as well as a test that would catch this, we'd be happy to merge it.